include/websock.js

```diff
@@ -321,26 +341,16 @@ function Websock() {
 
         // private methods
         _encode_message: function () {
-            if (this._mode === 'binary') {
-                // Put in a binary arraybuffer
-                return (new Uint8Array(this._sQ)).buffer;
-            } else {
-                // base64 encode
-                return Base64.encode(this._sQ);
-            }
+            // Put in a binary arraybuffer
+            // according to the spec, you can send ArrayBufferViews with the send method
```

Comment:
Yeah, I'm sure it hasn't been relevant for a long time. But doesn't hurt to make sure everywhere we are interested in supporting.